lib/registries/addon/drafts/draft/-components/tags-manager/template.hbs

```diff
@@ -0,0 +1,8 @@
+{{yield
+    (hash
+        addTag=(action this.addTag)
+        removeTag=(action this.removeTag)
+        clickTag=(action this.clickTag)
```

Comment:
No longer need to use action helper `(action ...)` since the `@action` decorator now does the binding.